be/src/olap/aggregate_func.h

```diff
@@ -203,6 +203,36 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_MIN, OLAP_FIELD_TYPE_LARGEINT>
     }
 };
 
+template <>
+struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_MIN, OLAP_FIELD_TYPE_VARCHAR> :
+        public AggregateFuncTraits<OLAP_FIELD_AGGREGATION_NONE, OLAP_FIELD_TYPE_VARCHAR> {
+    static void update(RowCursorCell* dst, const RowCursorCell& src, Arena* arena) {
+        bool src_null = src.is_null();
```

Comment:
I think is dst is null, you can just return, because null is the mimimum value.

Reply:
We want to ignore null for comparison. We want null to be always replaced.

Reply:
Oh, I see~